portal-survey-2023.Rmd

```diff
@@ -0,0 +1,260 @@
+These are community constituents who have significant say in how research should be done, e.g. how to both obtain and share data for a study.
+
+On the other hand, there are several caveats in the respondent profile that should be noted. 
+First is the relatively low percentage of bioinformaticians/computational biologists, which would be the demographic expected to be doing the practical, hands-on reuse of data. 
+This proportion for bioinformaticians/computational biologists is also about the same as a [previous RFC survey](https://nf-osi.github.io/research/rfc-brief.html#respondent-profiles). 
+A data portal should arguably see *over-representation* of bioinformaticians in its userbase, suggesting that the Portal either needs to have more bioinformatician users or increase survey engagement for this group. 
+<!-- read: computational people may fill out surveys at a lower rate -->
+The second issue is that the survey question did not adequately admit additional demographics. 
+One other type of user that should perhaps have been captured is the "rare disease patient or patient advocate". 
+In further research and follow-up, we did see that a small minority of respondents fit into this demographic, though this is not apparent in the official responses because of the way the question was structured.
+
+```{r reshape, echo=FALSE}
+
+# long format (tidy) data needed for subsequent figures
+
+tidy_survey_quant <- survey %>% 
+  select(starts_with("How would you generally rate your experience")) %>% 
+  tidyr::pivot_longer(cols = !id, names_to = "area", values_to = "rating")
+
+area_regex <- "(ability to \\w+)"
+issue_regex <-"(?<=\\[).+?(?=\\])"
+
+tidy_survey_ordinal <- survey %>% 
+  select(contains("sort")) %>%
+  tidyr::pivot_longer(cols = !id, names_to = "area_issue", values_to = "response") %>%
+  mutate(area = regmatches(area_issue, regexpr(area_regex, area_issue, perl = T)),
+         issue = regmatches(area_issue, regexpr(issue_regex, area_issue, perl = T)))
+
+
+grouped_bar <- function(data) {
+  
+  ggplot(data, aes(x = issue, y = n, fill = response)) +
+       geom_bar(stat="identity") +
+  scale_fill_manual(values = c("#E64B35FF",  "orange", "#4DBBD5FF", "gray")) +
+  theme_minimal() +
+  xlab("") +
+  ylab("Number of responses") +
+  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust=1),
+        plot.margin = margin(3, 1, 0.4, 1, "cm")) # need generous left margin for long text
+}
+
+```
+
+### Data findability
+
+Data discovery is an important antecedent to data reuse. 
+The first survey component asked about issues community users might encounter when trying to **find** data through the NF Data Portal. 
+ 
+Of several issues that might hamper findability, Figure \@ref(fig:fig-findability) indicates that a confusing user interface had the most severe effect, though poor search functionality was also detrimental. 
```

Comment:
Too verbose

Reply:
@anngvu Since there is a reason behind making this section more verbose, I thought I will lay out the rationale here. Here I have parsed out two different observations from the analysis and plots taken together. From the next sentence and the plots, it looked like almost half or greater than half of the respondents had mentioned that finding data was not an issue for them. This observation does not come through in the analysis and discussion as written currently. 
Additionally, I think adding quants more explicitly is important for any analysis. So I suggested adding actual numbers here. 
Of course these are just suggestions, so please feel free to take it or leave it as you prefer.